ding/reward_model/rnd_reward_model.py

```diff
@@ -156,3 +169,14 @@ def collect_data(self, data: list) -> None:
 
     def clear_data(self) -> None:
         self.train_obs.clear()
+
+    def reward_deepcopy(self, train_data):
```

Comment:
call `reward_deepcopy` method in `estimate` method by default, and remove deepcopy in `serial_pipeline_xxx`